packages/docs/content/components/alert.mdx

```diff
@@ -182,19 +182,46 @@ Used to show important but not as urgent messages as standard alerts.
 
 ### Accessibility
 
+- Use alerts sparingly as to avoid creating too much “noise” in screen readers, making it difficult to understand the overall layout of the page.
+- Users who are viewing the content need it to be placed in an effective location and to draw attention even when CSS is turned off. Content can help here too, using word choices that make the importance of the message clear.
+- Users who hear this content through assistive technology, such as a screen reader, will need to be alerted at the right time, and that can be managed with code.
+- Do not visually hide alert messages on the page and then make them visible when they are needed. Users of older assistive technologies may still be able to perceive the alert messages even if they are not currently applicable.
+- Use the <abbr title="Accessible rich internet applications">ARIA</abbr> `role="alert"` to inform assistive technologies of a time-sensitive and important message that is not interactive. If the message is interactive, use the `role="alertdialog"` instead. Both include an implicit use of `aria-live=”assertive”`. Refer to the [Dialog component’s full documentation](/components/modal-dialog/) when using `role=”alertdialog”`.
+- If you have multiple alert components on a page, tailor the accessible text to differentiate between each by using an `aria-label` or `aria-labelledby` attribute.
+
 <ThemeContent onlyThemes={['healthcare']}>
 
-If dynamically loaded, be mindful of when the alert displays on the page. For example, an alert could appear on a page due to an error or progressive disclosure. Where and how the content appears visually should align with how and when a screen reader user hears the content. Errors need immediate attention, but content through progressive disclosure may not.
+**For dynamic alerts**
```

Comment:
Should this be a heading? It's too deep to show up in the table of contents, the fact that it's only for healthcare might not be a problem.